Dockerfile

```diff
@@ -4,41 +4,22 @@ FROM python:3.8-alpine
 ENV USER="app"
 ENV GROUP="app"
 ENV HOME="/home/$USER"
-ENV APP_HOME="$HOME/web"
+ENV PROJECT_HOME="$HOME/web"
 ENV PATH="$HOME/env/bin:$PATH"
 ENV VIRTUAL_ENV="$HOME/env"
 
 # env - python tweeking
 ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONUNBUFFERED=1
 
-# env - app
-ENV DEBUG=1
-ENV DJANGO_SUPERUSER_USERNAME="juntagrico"
-ENV DJANGO_SUPERUSER_PASSWORD="juntagrico"
-ENV DJANGO_SUPERUSER_EMAIL="juntagrico@localhost.localhost"
-ENV JUNTAGRICO_SECRET_KEY="juntagrico"
```

Comment:
Not sure if we should set a default here: https://github.com/Rote-Beete/juntagrico-docker/blob/main/settings.py#L11

Reply:
There is no sane default for a secret, because a secret should not be reused.

Best common practice is for an image to fail if no secret is provided.